src/ramses_tx/schemas.py

```diff
@@ -232,7 +232,7 @@ def convert_null_to_dict(node_value: _T | None) -> _T | dict[Never, Never]:
 SCH_DEVICE_ID_UFC = vol.Match(DEVICE_ID_REGEX.UFC)
 
 _SCH_TRAITS_DOMAINS = ("heat", "hvac")
-_SCH_TRAITS_HVAC_SCHEMES = ("itho", "nuaire", "orcon")
+_SCH_TRAITS_HVAC_SCHEMES = ("climarad", "itho", "nuaire", "orcon", "vasco")
```

Comment:
This is not part of this PR, it should be with `22F1`: please remove it from this PR.

BTW, climarad / vasco use the same scheme? If so, call it vasco.

Reply:
ClimaRad used the vasco scheme on their MiniBox fan, but a different one on the Ventura HRU (22F4). Will split them up

Reply:
As far as I can recall (and I wrote most of the code), schemes only apply to `22F1`.

I have no idea what a `22F4` is for - I'd love your input on that.

Reply:
ClimaRad doesn't share their list, but their VenturaV1x HRU uses 22F4 for speed steps, while others use 22F1.
So our code needs a scheme for that, and I would expect that scheme (as well as this list of scheme names) to live close to the existing ones. But I will wait till I work on 22F4.